### PR TITLE
Remove references to Notify team, in favour of Emergency Alerts

### PR DESCRIPTION
**Pull request by ChrisBAshton**
The Emergency Alerts service is now managed by the Emergency Alerts team.

### CODE REVIEW

File: source/manual/emergency-alerts.html.md [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
+title: What are Emergency Alerts?
 layout: manual_layout
 type: learn
 parent: "/manual.html"
-section: Notify Emergency Alerts
+section: Emergency Alerts
 important: true
 ---
 
 The Cabinet Office and the Department for Digital, Culture, Media & Sport are developing a ‘cell broadcast’ alert system to enable people whose lives are at risk in an emergency to be rapidly contacted via their mobile phone. They are currently at the testing phase and subject to successful progress, hope to launch the service soon.
```
Comment (nicholsj) [inline source/manual/emergency-alerts.html.md:11]:
```suggestion
The Cabinet Office and the Department for Digital, Culture, Media & Sport have developed a ‘cell broadcast’ alert system to enable people whose lives are at risk in an emergency to be rapidly contacted via their mobile phone. 
```

File: source/manual/emergency-alerts.html.md [hunk 2]
```diff
@@ -30,10 +30,10 @@ We [only have one backend](https://github.com/alphagov/govuk-fastly-secrets/blob
 API Keys
 --------
 
-Notify have three Fastly API keys with [`purge_select` scope](https://developer.fastly.com/reference/api/auth/#scopes), one for
+Emergency Alerts have three Fastly API keys with [`purge_select` scope](https://developer.fastly.com/reference/api/auth/#scopes), one for
 integration, staging and production. These allow them to purge individual pages or surrogate keys from the cache. Note that the scope
-of these keys is not restricted to `/alerts` - in principle they could be used to purge any page from the cache. We trust Notify to take
+of these keys is not restricted to `/alerts` - in principle they could be used to purge any page from the cache. We trust Emergency Alerts to take
```
Comment (nicholsj) [inline source/manual/emergency-alerts.html.md:35]:
```suggestion
of these keys is not restricted to `/alerts` - in principle they could be used to purge any page from the cache. We trust the Emergency Alerts team to take
```

File: source/manual/emergency-alerts.html.md [hunk 3]
```diff
@@ -30,10 +30,10 @@ We [only have one backend](https://github.com/alphagov/govuk-fastly-secrets/blob
 API Keys
 --------
 
-Notify have three Fastly API keys with [`purge_select` scope](https://developer.fastly.com/reference/api/auth/#scopes), one for
+Emergency Alerts have three Fastly API keys with [`purge_select` scope](https://developer.fastly.com/reference/api/auth/#scopes), one for
```
Comment (nicholsj) [inline source/manual/emergency-alerts.html.md:33]:
```suggestion
The Emergency Alerts team have three Fastly API keys with [`purge_select` scope](https://developer.fastly.com/reference/api/auth/#scopes), one for
```

**Review by nicholsj — CHANGES_REQUESTED** [review]
Looks good, thanks. Some minor suggested changes for clarity

**Comment by ChrisBAshton** [comment]
Superseded by #4301.

**Pull request closed without merging**